src/response.cpp

```diff
@@ -52,8 +52,8 @@ bool HTTP::CHTTPResponse::LoadHeaders(size_t &nIndex, const std::vector<char> &c
 	bool bIsNewLine{false};
 	bool bIsReadingValue{false};
 
-	std::string sReadKey;
-	std::string sReadValue;
+	std::string sReadKey{""};
+	std::string sReadValue{""};
```

Comment:
String has ctor. You don't need to initialize it with empty data

Reply:
Removed redundant constructor